src/app/components/resources/resources.component.html

```diff
@@ -178,15 +175,24 @@ <h6 class="dropdown-header">OTHER</h6>
             Alphabetical
           </button>
           <button
-            (click)="updateSort('default')"
+            (click)="updateSort('tool')"
             class="filter-dropdown-button"
-            [ngClass]="sortOrder == 'default' ? 'bg-secondary text-light' : ''"
+            [ngClass]="sortOrder == 'tool' ? 'bg-secondary text-light' : ''"
           >
-            Default Order
+            Tool Order
```

Comment:
Have you been asked to change this name from the God Tools team? If not, I would set it back to what it was.

Reply:
I wasn't asked but I think it makes it more clear, like you mentioned in your earlier comment. I will ask them what they prefer.